File: code/code-interface/src/main/java/cn/moonnow/code/query/DtQuery.java
package cn.moonnow.code.query;

import java.io.Serializable;
import java.util.List;

import lombok.Data;

/**
 * 数据库表 PRO_DT 查询类
 */
@Data
public class DtQuery implements Serializable {

  private static final long serialVersionUID = 1L;

  protected String dtId; // 数据库表编号

  protected String dtSql; // 数据库表sql

  protected String dtName; // 表名

  protected String dtNameAnnotation; // 表名注释

  protected String dtPrefix; // 数据库表前缀

  protected String initialCaseEntityName; // 首字母大写实体类名

  protected String initialLowercaseEntityName; // 首字母小写实体类名

  protected String proPath; // 项目路径

  protected String proAllName; // 项目全称

  protected List<String> dtIdAndin; // 数据库表编号Andin查询

  protected String dtNameAndeq; // 表名Andeq查询

  protected String dtNameAndKeyLike; // 表名AndKeyLike查询

  protected String dtNameAnnotationOrKeyLike; // 表名注释OrKeyLike查询

  protected String dtPrefixOrKeyLike; // 数据库表前缀OrKeyLike查询

  protected String initialCaseEntityNameOrKeyLike; // 首字母大写实体类名OrKeyLike查询

  protected String initialLowercaseEntityNameOrKeyLike; // 首字母小写实体类名OrKeyLike查询

  protected String proPathOrKeyLike; // 项目路径OrKeyLike查询

  protected String proAllNameOrKeyLike; // 项目全称OrKeyLike查询

}

File: vue-dev-tool/src/view/cn/moonnow/code/dt/DtQuery.js
export default class DtQuery {
  constructor () {
    this.dtId = null // 数据库表编号
    this.dtSql = null // 数据库表sql
    this.dtName = null // 表名
    this.dtNameAnnotation = null // 表名注释
    this.dtPrefix = null // 数据库表前缀
    this.initialCaseEntityName = null // 首字母大写实体类名
    this.initialLowercaseEntityName = null // 首字母小写实体类名
    this.proPath = null // 项目路径
    this.proAllName = null // 项目全称
    this.dtIdAndin = null // 数据库表编号Andin查询
    this.dtNameAndeq = null // 表名Andeq查询
    this.dtNameAndKeyLike = null // 表名AndKeyLike查询
    this.dtNameAnnotationOrKeyLike = null // 表名注释OrKeyLike查询
    this.dtPrefixOrKeyLike = null // 数据库表前缀OrKeyLike查询
    this.initialCaseEntityNameOrKeyLike = null // 首字母大写实体类名OrKeyLike查询
    this.initialLowercaseEntityNameOrKeyLike = null // 首字母小写实体类名OrKeyLike查询
    this.proPathOrKeyLike = null // 项目路径OrKeyLike查询
    this.proAllNameOrKeyLike = null // 项目全称OrKeyLike查询
  }
}
